RFS/src/main/java/com/rfs/common/SnapshotShardUnpacker.java

```diff
@@ -58,6 +60,35 @@ public void unpack(ShardMetadata.Data shardMetadata) {
         }
     }
 
+    public void cleanUp(ShardMetadata.Data shardMetadata) {
```

Comment:
Rather than have this method, lets have reuse the java convention to implement AutoCloseable so its clear that when you are done with an unpacker it should clean up itself.

This allows code like:

```java
try (var unpacker = getShardUnpacker()) {
   unpack.things();
}
// unpacker close method was automatically called here
```